README.md

```diff
@@ -344,6 +344,29 @@ Hyperwallet.shared.listPrepaidCards(queryParam: prepaidCardQueryParam) { (result
 }
 ```
 
+### List Prepaid Card Receipts
+```swift
+let receiptQueryParam = HyperwalletReceiptQueryParam()
+receiptQueryParam.createdAfter = ISO8601DateFormatter.ignoreTimeZone.date(from: "2016-12-01T00:00:00")
+receiptQueryParam.createdBefore = ISO8601DateFormatter.ignoreTimeZone.date(from: "2020-12-31T00:00:00")
```

Comment:
No need of setting createdBefore so you can remove this line

Reply:
Also add List User Receipts

Reply:
Removed unnecessary line. Added description for List User Receipts